vendor/bat-native-ledger/src/bat/ledger/internal/media/twitch.h

```diff
@@ -48,12 +48,12 @@ class MediaTwitch : public ledger::LedgerCallbackHandler {
   static std::string GetMediaURL(const std::string& mediaId);
 
   static std::string GetTwitchStatus(
-    const ledger::TwitchEventInfo& old_event,
-    const ledger::TwitchEventInfo& new_event);
+      const ledger::TwitchEventInfo& old_event,
```

Comment:
we should avoid doing this. If you want to do formatting do it in a separate PR as this introduce changes that are not related to this PR